docs/examples/wikitext.ts

```diff
@@ -17,56 +15,54 @@ async function main(): Promise<void> {
   const task = tasks.get('wikitext-103')
   if (task === undefined) { throw new Error('task not found') }
   
-  // Load the wikitext dataset from the `datasets` folder
-  const dataset = await loadWikitextData(task)
-
-  // Initialize a Disco instance and start training a language model
-  const aggregator = new aggregators.MeanAggregator()
-  const client = new clients.federated.FederatedClient(url, task, aggregator)
-  const trainingInformant = new informant.FederatedInformant(task, 10)
-  const disco = new Disco(task, { scheme: 'federated', client, aggregator, informant: trainingInformant })
-  await disco.fit(dataset)
-
-  // Get the model and complete the prompt
-  if (aggregator.model === undefined) {
-    throw new Error('model was not set')
-  }
-  const model = aggregator.model as models.GPT
-
-  // Retrieve the tokenizer used during training
-  const tokenizer = await models.getTaskTokenizer(task)
-
-  const prompt = 'Hello world'
-  console.log(await model.generate(prompt, tokenizer))
-  
-  // Save the trained model
   const modelFolder = './models'
-  try {
-    if (!fs.existsSync(modelFolder)) {
-      fs.mkdirSync(modelFolder)
-    }
-  } catch (err) {
-    console.error(err);
-  }
-  const encoded = await serialization.model.encode(model)
-  await fsPromises.writeFile(`${modelFolder}/model.json`, encoded)
+  const modelFileName = 'model_random.json'
   
-  /**
-   * The commented code below shows how to load an existing model
-   */
-
-  // const content = await fsPromises.readFile(`${modelFolder}/model.json`)
-  // model = await serialization.model.decode(content) as models.GPT
-  // console.log(await model.generate(sample, tokenizerName, 10))
+  // Toggle TRAIN_MODEL to either train and save a new model from scratch or load an existing model
+  const TRAIN_MODEL = true
+  if (TRAIN_MODEL) {
```

Comment:
consider having a common code after this block, as single way to use the model, that makes it clearer IMO

Reply:
What do you mean? Like reverting to when loading a model was commented code rather than having a boolean constant?

Reply:
ho right, I wrote this comment quite in a hurry, let me reformulate:
it would be nice to show that training the model is optional. having a boolean is already a good way to do it, and to push it a bit further, having the same body of code at the end on both paths would be even nicer. smth like the following
```ts
// optionnaly train the model if wanted
if (TRAIN_MODEL) { ... }

// use saved model 
const model = ...
const prompt = ...
```

Reply:
Ah yes yes absolutely